welcome.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta content="text/html;charset=utf-8" http-equiv="Content-Type">
+    <meta content="utf-8" http-equiv="encoding">
+    <title>Welcome!</title>
+    <link rel="icon" type="image/ico" href="images/icon.png" />
+
+    <link rel="stylesheet" type="text/css" href="css/style.css">
+    <link rel="stylesheet" type="text/css" href="css/common.css">
+    <link rel="stylesheet" type="text/css" href="css/welcome.css">
+
+  </head>
+  <body>
+    <div id="container">
+      <div id="header">
+        <h1>Internet <img src="images/icon.png" alt="Internet Archive"> Archive</h1>
+      </div>
+      <div id="content">
+        <h1>Welcome!</h1>
+        <p>Here’s your brand new Internet Archive browser extension.</p>
+        <p>The extension collects and stores information about the search queries you make and the web pages you view to provide you with information about those pages and to help Internet Archive measure the Web. In some cases, information collected by the extension may be personally identifiable, but we do not attempt to analyze any information collected by the extension to determine the identity of any user.</p>
```

Comment:
You say "measure the Web"

Maybe we wanted to say "archive the web"?

Reply:
"Help the Internet Archive archive the web" doesn't sound great.  Maybe I'll replace IA with WBM in this case